RELEASE.md

```diff
@@ -1,3 +1,418 @@
+
+In order to reduce redundancy and overcomplexity in the CDM, a refactoring of the `OptionPayout` structure is required. The information contained in the fields inside the `optionStyle` (`americanExercise`, `europeanExercise`, and `bermudaExercise`) can be unified under a new type `ExerciseTerms`. This will reduce the redundancy of having the same information repeated under the three styles and improve the simplicity of the model. To distinguish whether the option is American, European, or Bermuda, a new `style` enumeration is added to the model inside `ExerciseTerms`, with the values "american", "european", and "bermuda". Additionally, the `strike` field, previously under `exerciseTerms`, is moved outside and located directly under `OptionPayout`, given that it does not convey any information about the exercise terms of an option.
+
+_What is being released?_
+
+- The `optionStyle` is removed from the model, along with the `americanExercise`, `europeanExercise`, and `bermudaExercise` fields.
+- A new `ExerciseTerms` structure is added to `OptionPayout`, containing all _distinct_ fields found previously under the three exercise styles.
+- A new `style` enumeration is added under `ExerciseTerms` to distinguish the style of the option. This enumeration is made **optional** to account the exercise terms of a `CancelableProvision`, `ExtendibleProvision`, or `OptionalEarlyTermination`, where the style is often derived from the structure itself.
+- The structures `CancelableProvision`, `ExtendibleProvision`, and `OptionalEarlyTermination` have been modified to use `ExerciseTerms` instead of the old `americanExercise`, `europeanExercise`, and `bermudaExercise` fields.
+- **Synonym mappings** have been modified to reflect these changes.
+
+_Data types_
+
+- Removed `OptionExercise` type.
+- Removed `OptionStyle` type.
+- Removed `AmericanExercise`, `EuropeanExercise`, and `BermudaExercise` types.
+- `strike` attribute of type `OptionStrike` added to `OptionPayout` type.
+- `exerciseTerms` attribute of type `ExerciseTerms` added to `OptionPayout` type.
+- `exerciseTerms` attribute of type `OptionExercise` removed from `OptionPayout` type.
+- `americanExercise`, `europeanExercise`, and `bermudaExercise` attributes of types `AmericanExercise`, `EuropeanExercise`, and `BermudaExercise` respectively removed from `CancelableProvision`, `ExtendibleProvision`, and `OptionalEarlyTermination` types.
+- `exerciseTerms` attribute of type `ExerciseTerms` added to `CancelableProvision`, `ExtendibleProvision`, and `OptionalEarlyTermination` types.
+-  Added new `ExerciseTerms` type with all of the distinct fields present before in `AmericanExercise`, `EuropeanExercise`, and `BermudaExercise` types.
+
+_Enumerations_
+
+- Added new `OptionExerciseStyleEnum` enumeration.
+
+- _Backward Incompatible Changes_
+
+
+- _Motivation for Change_
+
+This change modifies the `OptionPayout` structure to simplify it and reduce the redundancy and over-complexity of the model.
```

Comment:
Remove this "motivation for change" section given the "background" section in preamble.